kornia/feature/loftr/utils/supervision.py

```diff
@@ -46,9 +46,9 @@ def spvs_coarse(data, config):
 
     # 2. warp grids
     # create kpts in meshgrid and resize them to image resolution
-    grid_pt0_c = create_meshgrid(h0, w0, False, device).reshape(1, h0 * w0, 2).repeat(N, 1, 1)  # [N, hw, 2]
+    grid_pt0_c = create_meshgrid(h0, w0, False, device).reshape(1, h0 * w0, 2).expand(N, h0 * w0, 2)  # [N, hw, 2]
```

Comment:
```suggestion
    grid_pt0_c = create_meshgrid(h0, w0, False, device).expand(N, h0 * w0, 2)  # [N, hw, 2]
```
won't this work ?

Reply:
looks like it but I need to review carefully.
In general - as long as you don't plan to modify the resulting tensor (say by in-place assignment) you can expand it instead of repeating.

Reply:
@edgarriba there is a later assignment in the case you shared... I suggest saving it to a different PR